### DISCUSSION TITLE
propagate option on resave via console

### DISCUSSION THREAD

**Discussion by vnali** [craftcms/cms#11032] [Q&A] [answered]
i see a [propagate option on console resave controller](https://github.com/craftcms/cms/blob/ba35db6ba243d9c294e3e302150b5730f1c417c0/src/console/controllers/ResaveController.php#L169) but it seems it is not used on resave controller at all. actually i can't understand how propagate option is going to affect resave behavior. maybe this is related to autoResaveEntries?

**Comment by brandonkelly** [comment, marked as answer]
Looks like that was never hooked up to anything. Just removed it for the next release (3083d453c236865287de81d131d01e503107fdbb).

**Reply by brandonkelly** [reply]
3.7.40 is out now with that fix.
